src/data/cognito.ts

```diff
@@ -0,0 +1,33 @@
+import { CognitoIdentityServiceProvider } from 'aws-sdk';
+import { Auth } from 'aws-amplify';
+import {
+    ListUserPoolsRequest,
+    ListUsersRequest,
+    UserPoolDescriptionType,
+    UserType,
+} from 'aws-sdk/clients/cognitoidentityserviceprovider';
+import { AWS_REGION } from '../constants';
+
+export const getCognitoClient = async (): Promise<CognitoIdentityServiceProvider> =>
+    new CognitoIdentityServiceProvider({ region: AWS_REGION, credentials: await Auth.currentUserCredentials() });
+
+export const getUserPoolList = async (cognito: CognitoIdentityServiceProvider): Promise<UserPoolDescriptionType[]> => {
+    const params: ListUserPoolsRequest = { MaxResults: 2 };
```

Comment:
Why is MaxResults this set to 2? If we added more user pools, could this lead to issues?

Reply:
Yep, fair point, I don't think we should need anymore but there's no reason not to give a bit of headroom